pkg/webhook/parser.go

```diff
@@ -70,9 +70,15 @@ type Config struct {
 	// Versions specifies the API versions that this webhook receives requests for.
 	Versions []string
 
-	// Name indicates the name of this webhook configuration.
+	// Name indicates the name of this webhook configuration. Should be a domain with at least three segments separated by dots
```

Comment:
> Should be a domain with at least three segments separated by dots

Probably just say:
`Name should be fully qualified, e.g. imagepolicy.kubebuilder.io`.

Reply:
An [FQDN](https://tools.ietf.org/html/rfc1594#section-5) can also simply be `kubebuilder.io`, so this is more specific about the requirements I think. This is simply using the wording direct from the webhook validation. This is the error you receive if you try to apply a webhook with a name with < 3 labels:

>The ValidatingWebhookConfiguration "validating-webhook-configuration" is invalid: webhooks[0].name: Invalid value: "funfunfun": should be a domain with at least three segments separated by dots